framework/PageForm/Inputs/PageFormGroup.tsx

```diff
@@ -1,15 +1,32 @@
-import { Button, FormGroup, FormGroupProps, Popover } from '@patternfly/react-core';
+import { Button, FormGroup as PFFormGroup, FormGroupProps, Popover } from '@patternfly/react-core';
 import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
-import { ReactNode } from 'react';
+import { ReactElement } from 'react';
+import styled from 'styled-components';
 
 export type PageFormGroupProps = Pick<
   FormGroupProps,
-  'children' | 'helperText' | 'helperTextInvalid' | 'isRequired' | 'label'
-> & { id?: string; labelHelpTitle?: string; labelHelp?: ReactNode };
+  'children' | 'helperText' | 'helperTextInvalid' | 'isRequired' | 'label' | 'labelIcon'
+> & {
+  id?: string;
+  labelHelpTitle?: string;
+  labelHelp?: string;
+  additionalControls?: ReactElement;
+};
+
+const FormGroup = styled(PFFormGroup)`
+  ${(props: PageFormGroupProps) =>
+    !props.labelIcon &&
+    `{
+      .pf-c-form__group-label {
+        padding-bottom: 20px
```

Comment:
```suggestion
        padding-bottom: 1.25rem;
```

This allows it to stay aligned if the base font size is adjusted

Reply:
...actually, thinking about it further, I think the issue is more that we need to make the icon button shorter/reduce its padding